src/lib/utils/hooks/SynapseAPI/forum/useForum.ts

```diff
@@ -205,30 +207,43 @@ export function useCreateThread(
       SynapseClient.postThread(accessToken, newThread),
     {
       ...options,
-      onSuccess: async (newThread, variables, ctx) => {
-        await queryClient.invalidateQueries(['forumthread', newThread.forumId])
+      onSuccess: async (threadBundle, newThreadRequest, ctx) => {
+        await queryClient.invalidateQueries([
+          'forumthread',
+          threadBundle.forumId,
+        ])
+        await queryClient.invalidateQueries(['thread', threadBundle.id])
```

Comment:
Is this line necessary? The thread is new so we shouldn't have any information about its id in the cache.
```suggestion
```